app/src/main/java/com/example/cachua/musicdemo21/ui/music/adapter/MusicAdapter.kt

```diff
@@ -0,0 +1,70 @@
+package com.example.cachua.musicdemo21.ui.music.adapter
+
+
+import android.content.Context
+
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import android.support.constraint.ConstraintLayout
+import android.widget.ImageView
+import com.bumptech.glide.Glide
+import com.example.cachua.musicdemo21.R
+import com.example.cachua.musicdemo21.data.model.MusicModel
+
+
+class MusicAdapter (var mContext: Context,var mMusicArray: ArrayList<MusicModel>): RecyclerView.Adapter<MusicAdapter.ViewHolder>(){
+
+    private lateinit var callBack: CallBack
+
+    override fun onCreateViewHolder(p0: ViewGroup, p1: Int): ViewHolder {
+        val layoutInflater = LayoutInflater.from(p0.context).inflate(R.layout.item_layout, p0, false)
+        return ViewHolder(layoutInflater)
+    }
+
+    override fun getItemCount(): Int {
+        return mMusicArray.size
+    }
+
+    override fun onBindViewHolder(p0: ViewHolder, p1: Int) {
+        var music= mMusicArray[p1]
+        p0.mTextTitlePlay.text= music.title
```

Comment:
```suggestion
        viewHolder.bindData(music)
```
Đổi tên các params cho dễ đọc, dễ hình dùng hơn.